src/data/camera.ts

```diff
@@ -38,22 +38,25 @@ export const computeMJPEGStreamUrl = (entity: CameraEntity) =>
 
 export const fetchThumbnailUrlWithCache = (
   hass: HomeAssistant,
-  entityId: string
+  entityId: string,
+  ...args: any[]
```

Comment:
since this is a thumbnail specific method, we should not use `args` but instead use `width` and `height`

```suggestion
  width: number,
  height: number
```